laceworksdk/api/v2/vulnerabilities.py

```diff
@@ -90,6 +91,33 @@ def status(self, request_id):
             )
 
             return response.json()
+    
+    class ImageSummaryVulnerabilitiesAPI(SearchEndpoint):
+        """A class used to represent the Container Vulnerabilities API endpoint."""
+
+        RESOURCE = "ImageSummary"
+
+        def search(self, **request_params):
+            """A method to issue Container Vulnerability scans.
```

Comment:
Is this true? is this triggering a scan?

Reply:
resolved